docs/lua-records/index.rst

```diff
@@ -177,9 +177,10 @@ outside of Europe will hit 198.51.100.1 as long as it is available, and the
 
 Advanced topics
 ---------------
-By default, LUA records are executed with ``return `` prefixed to them. This saves
-a lot of typing for common cases. To run actual Lua scripts, start a record with a ``;``
-which indicates no ``return `` should be prepended.
+
+By default, LUA records are executed as if they were the argument to Lua's ``return`` statement.
+This saves a lot of typing for common cases.
+To run actual Lua scripts, start a record with a semicolon (``;``). You need to add your own ``return`` keyword to the last statement.
```

Comment:
In many scripts, it might not be the last statement (although the return is always the last statement executed), but anybody that goes down that path knows this.

Reply:
I've dropped "keyword to the last", then its not wrong.